### PR TITLE
NEXUS-7820 Show persistent message after session timeout

### PR DESCRIPTION
**Pull request by djsauble**
Makes the “expire session” modal persist, and adds a convenient sign in button.

Issue: https://issues.sonatype.org/browse/NEXUS-7820

Before: http://monosnap.com/file/ryQvrJ22GCfDFXLEoQ5FN8f0ZauC2K

After: http://monosnap.com/file/8FuvqsjcI8fB3thQZxpwu6Bj8X6ys4

### CODE REVIEW

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/UiSessionTimeout.js [hunk 1]
```diff
@@ -155,7 +155,10 @@ Ext.define('NX.controller.UiSessionTimeout', {
       run: function (count) {
         win.down('label').setText('Session will expire in ' + (me.SECONDS_TO_EXPIRE - count) + ' seconds');
         if (count === me.SECONDS_TO_EXPIRE) {
-          win.close();
+          win.down('label').setText('Your session has expired. Please log in.');
```
Comment (kellyrob99) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/UiSessionTimeout.js]: NX.I18n?

**Comment by kellyrob99** [comment]
+1

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/UiSessionTimeout.js (hunk 1 above)
Comment (djsauble) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/UiSessionTimeout.js]: Wanted to get the other I18n-related PRs merged before tackling this. Will do before merge.

/me hates merge conflicts

Comment (djsauble) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/UiSessionTimeout.js]: Done.

**Comment by djsauble** [comment]
Manual merge. https://github.com/sonatype/nexus-oss/commit/b42e60267829d1ec104dc8eda193ae6f51d4a5d7

**Pull request closed without merging**